pandas/core/indexes/base.py

```diff
@@ -681,7 +681,47 @@ def _values(self):
         return self.values
 
     def get_values(self):
-        """ return the underlying data as an ndarray """
+        """
+        Return `Index` data as an `numpy.ndarray`.
+
+        It is a getter wrapper around `Index.values`.
```

Comment:
Can rmeove this, since it's repeated in the See Also (don't worry about the warning).